tailwind.config.js

```diff
@@ -43,24 +43,24 @@ module.exports = {
         'pulse-horizontal': 'pulse-horizontal 2s ease-in-out infinite',
       },
       colors: {
-        'blue-900': '#0d0030',
-        'blue-800': '#1a0259',
-        'blue-700': '#3a00d6',
-        'blue-600': '#4b61d5',
-        'blue-500': '#15a1fc',
-        'blue-400': '#bff5fa',
+        accent: '#15a1fc',
+        'accent-dark': '#0d0030',
```

Comment:
I am not sure if the blue is `accent`, it's mostly used on the background. It more feels like `primary`

Reply:
Changed them